backend/tests/db/test_language.py

```diff
@@ -39,68 +38,92 @@ def test_get_languages_single(dbsession: OrmSession, english_schedule: Schedule)
-        language_code="eng",
-        language_name_en="English",
-        language_name_native="English",
-        name="test",
-        category="test",
-        config={"test": "test"},
-        enabled=True,
-        tags=["test"],
-        periodicity="test",
-        notification={"test": "test"},
-    )
-    schedule2 = Schedule(
-        language_code="eng",
-        language_name_en="English (US)",
-        language_name_native="English (US)",
-        name="test2",
-        category="test",
-        config={"test": "test"},
-        enabled=True,
-        tags=["test"],
-        periodicity="test",
-        notification={"test": "test"},
-    )
-    dbsession.add_all([schedule1, schedule2])
-    dbsession.flush()
-
-    # Get languages - should only return one entry due to distinct
-    results = get_languages(dbsession, skip=0, limit=100)
-    assert results.nb_languages == 1
-    assert len(results.languages) == 1
+    # The name should come from pycountry, not the database
+    assert results.languages[0].name == "English"
 
 
 def test_get_languages_pagination(dbsession: OrmSession):
     """Test getting languages with pagination."""
-    # Create test schedules
+    # Create test schedules with valid language codes
     schedules = [
         Schedule(
-            language_code=f"lg{i}",
-            language_name_en=f"Language {i}",
-            language_name_native=f"Language {i}",
-            name=f"test{i}",
+            language_code="eng",  # English
+            name="test1",
+            category="test",
+            config={"test": "test"},
+            enabled=True,
+            tags=["test"],
+            periodicity="test",
+            notification={"test": "test"},
+        ),
+        Schedule(
+            language_code="fra",  # French
+            name="test2",
+            category="test",
+            config={"test": "test"},
+            enabled=True,
+            tags=["test"],
+            periodicity="test",
+            notification={"test": "test"},
+        ),
+        Schedule(
+            language_code="spa",  # Spanish
+            name="test3",
+            category="test",
+            config={"test": "test"},
+            enabled=True,
+            tags=["test"],
+            periodicity="test",
+            notification={"test": "test"},
+        ),
+        Schedule(
+            language_code="eng",  # English
+            name="test4",
+            category="test",
+            config={"test": "test"},
+            enabled=True,
+            tags=["test"],
+            periodicity="test",
+            notification={"test": "test"},
+        ),
+        Schedule(
+            language_code="spa",  # Spanish
+            name="test5",
             category="test",
             config={"test": "test"},
             enabled=True,
             tags=["test"],
             periodicity="test",
             notification={"test": "test"},
-        )
-        for i in range(5)
+        ),
```

Comment:
You should add an invalid language code to check behavior (we should get the list of all other codes and nothing more)